repos/spack_repo/builtin/packages/intel_oneapi_compilers_classic/package.py

```diff
@@ -68,19 +66,26 @@ def compiler_version_regex(self):
         "2021.8.0": "2023.0.0",
         "2021.9.0": "2023.1.0",
         "2021.10.0": "2023.2.4",
-        "2021.11.1": "2024.0.2",
-    }.items():
+        "2021.11.0": "2024.0.0",
+        "2021.11.1": "2024.0.1:2024.0.2",
+        "2021.12.0": "2024.1.0",
+        "2021.13.0": "2024.2.0",
+        "2021.13.1": "2024.2.1",
```

Comment:
I thought we had to stop updating this package when they stopped shipping icc because a toolchain without a C compiler would not be useful. I am unable to try it out, but do you know what happens?

Reply:
Yeah, the `ifort` is still usable and I work with a few code teams that are stuck with `ifort` due to `ifx` either still not compiling their code or being sufficiently weak on performance for their code that it's non-viable.